lib/fog/openstack/compute.rb

```diff
@@ -6,10 +6,11 @@ class OpenStack < Fog::Service
       requires :openstack_auth_url
       recognizes :openstack_auth_token, :openstack_management_url,
                  :persistent, :openstack_service_type, :openstack_service_name,
-                 :openstack_tenant,
+                 :openstack_tenant, :openstack_tenant_id,
                  :openstack_api_key, :openstack_username, :openstack_identity_endpoint,
                  :current_user, :current_tenant, :openstack_region,
-                 :openstack_endpoint_type
+                 :openstack_endpoint_type,
+                 :openstack_prj_domain, :openstack_user_domain
```

Comment:
While I don't like excessive typing, what I don't like more is odd ball abbreviations I cannot remember.  Can we get openstack_project_domain?